module3/module3.qmd

```diff
@@ -16,15 +16,17 @@ title: "SESSION 3 - Introduction to Trimming, Assembly and QC"
 
 *Note: This exercise is a work of fiction. Any resemblance to a real life situation is purely coincidental.*
 
-In December 2020, three cases of primary bloodstream infections sustained by ertapenem-resistant *Klebsiella pneumoniae* were observed over a two-month period in the neonatal intensive care unit (NICU) at the University College Hospital in Ibadan, Oyo State. The NICU consists of a 15-bed tertiary level ward. Faecal screening of all admitted neonates was performed. Ten additional samples from the environment were collected. A plan of the NICU and sample collection can be seen below.
+In December 2020, three cases of primary bloodstream infections sustained by ertapenem-resistant *Klebsiella pneumoniae* were observed over a two-month period in the neonatal intensive care unit (NICU) at the University College Hospital in Ibadan, Oyo State. This NICU consists of a 15-bed tertiary level ward. Since the detectionf of these infections, faecal screening of all admitted neonates was performed. Additionally, ten samples from the environment were collected. The epidemiological and demographic data is summarised in the file [epi_data.csv](data/epi_data.csv).
 
-<img src="img/unit.png" width="70%" height="70%"/>
+<img src="img/unit.png" width="70%" height="70%"/><br/>
+Plan of the NICU. Locations of sample collection is indicated with light blue circles.
+<br/><br/>
 
-The reference lab has sent you the culture results in an Excel file called ([`lab_results.xlsx`](data/lab_results.xlsx)). Faecal screening of all admitted neonates, revealed that **five other neonates** were carriers of *K. pneumoniae*. The lab reported that **2 out of 10** environmental samples were positive for *K. pneumoniae*. **This immediately prompted the closure of 1 room.**
+The reference lab sent you the culture results from the faecal screening and the environmental sampling in an Excel file called ([`lab_results.xlsx`](data/lab_results.xlsx)). Faecal screening of the admitted neonates, revealed that **five other neonates** were carriers of *K. pneumoniae*. Also, the lab reported that **2 out of 10** environmental samples were positive for *K. pneumoniae*. **This immediately prompted the closure of 1 room.**
```

Comment:
not only faecal screening and environmental samples, the bloodstream strains as well.